src/stylesheets/base/_vars.scss

```diff
@@ -0,0 +1,2 @@
+$florence-blue: #184d66;
+$florence-orange: #c06d3a;
```

Comment:
this _vars files would be a good place for breakpoints, sizes, fonts, etc (maybe, if you are into that sort of thing)